plugins/coverbrowser.koplugin/mosaicmenu.lua

```diff
@@ -447,7 +447,7 @@ function MosaicMenuItem:update()
     local max_img_w = dimen.w - 2*border_size
     local max_img_h = dimen.h - 2*border_size
     local cover_specs = {
-        sizetag = "M",
+        sizetag = max_img_w .. max_img_h,
```

Comment:
Probably safer to keep it as a string with some letters :)
so there's no risk of collision when w=123 h=45 and when w=12 h=345.
`M123x45` would avoid that, while staying readable, and you could check in listmenu that it does start with `M` to not replace it.
https://github.com/koreader/koreader/blob/bf59f53e18125ce92f73980d254b2cac911f9c61/plugins/coverbrowser.koplugin/listmenu.lua#L273

And may be try to precompute it if possible like in listmenu.lua:
https://github.com/koreader/koreader/blob/bf59f53e18125ce92f73980d254b2cac911f9c61/plugins/coverbrowser.koplugin/listmenu.lua#L944
(which should probably use a size insted of this number?)

Reply:
I supposed to use the same approach to the List view as well. There is a big difference in thumbnails sizes for 4 and 30 (our new max) items per page.
So, to refresh the cache on every change of items per page.